Brick-Studios/BeastArena
# Added sprites

# Description

Added:

- Elephant
- Cheetah
- Rifle
- Sniper rifle
- Changed mass/firerate/speed for every different character/weapon
- Slightly changed gorilla

( Weapon position and bullet position will be changed in a different pull request)

## Screenshot(s)
![image](https://user-images.githubusercontent.com/33418574/69484310-11eb2c00-0e32-11ea-824c-3168e585fd72.png)

# How can this be tested?

- Can you see the elephant and cheetah?
- They both have different physics (mass), can you feel the difference?

- Can you see the rifle and sniper rifle?

Rifle:
- More damage
- Faster bullet
- Faster fire rate

Sniper:
- A lot more damage
- A lot faster bullet
- Slower fire rate

# Checklist:
